concepticondata/references/references.bib

```diff
@@ -4848,3 +4848,13 @@ @Article{Gruehn2008
   Year                                = {2008},
   Doi                                 = {10.3758/BRM.40.4.1088}
 }
+
+@Article{DellaRosa2010,
+  Title                               = {Beyond the abstract—concrete dichotomy: Mode of acquisition, concreteness, imageability, familiarity, age of acquisition, context availability, and abstractness norms for a set of 417 Italian words},
+  Author                              = {Della Rosa, Pasquale A. and Catrical{\`a}, Eleonora and Vigliocco, Gabriella and Cappa, Stefano F.},
+  Journal                             = {Behavior research methods},
```

Comment:
Just a minimal comment and question: Maybe we can put this in capital letters (Behavior Research Methods). Should we also add the Publisher = {Springer Nature}?

Reply:
Yes, capitalize Behavior Research Methods and publisher: Springer, as in e.g. Schroeder2012.

Reply:
In this case, the bibtex entry before to this one may be added the publisher as well :)